common/persistence/visibility/store/query/converter.go

```diff
@@ -479,24 +479,36 @@ func (c *comparisonExprConverter) Convert(expr sqlparser.Expr) (elastic.Query, e
 
 	var query elastic.Query
 	switch comparisonExpr.Operator {
-	case ">=":
+	case sqlparser.GreaterEqualStr:
 		query = elastic.NewRangeQuery(colName).Gte(colValues[0])
-	case "<=":
+	case sqlparser.LessEqualStr:
 		query = elastic.NewRangeQuery(colName).Lte(colValues[0])
-	case ">":
+	case sqlparser.GreaterThanStr:
 		query = elastic.NewRangeQuery(colName).Gt(colValues[0])
-	case "<":
+	case sqlparser.LessThanStr:
 		query = elastic.NewRangeQuery(colName).Lt(colValues[0])
-	case "=", "like": // The only difference is that "%" is removed for LIKE queries.
+	case sqlparser.EqualStr, sqlparser.LikeStr: // The only difference is that "%" is removed for LIKE queries.
```

Comment:
If `START_WITH` is just an alias to `LIKE` you should allow it too and make them work same way.
Let's not just remove `%` from the operand but fail query if `%` is anywhere but lust symbol.

Reply:
I will remove the LIKE support completely soon.

Reply:
Why are we removing LIKE support? Have we documented our rationale

Reply:
It doesn't behave as expected, and we don't have plans to support it in its entirety.